### PR TITLE
fix: secrets detection

### PR DESCRIPTION
**Pull request by pvinis**
This PR resolves []

### Description

Context: https://artsy.slack.com/archives/C02BAQ5K7/p1656407038938159

In short, we were using our own brew version of detect-secrets because the docker image we use had an older version. We don't need that anymore.

We now use the fresh python version of detect-secrets, and in another PR we will replace the docker image, so we don't have issues in the future.

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: Brewfile [hunk 1]
```diff
@@ -1,2 +1 @@
-brew "artsy/formulas/detect_secrets" # using brew version. read more here: https://github.com/artsy/homebrew-formulas/pull/13
```
Comment (pvinis) [inline Brewfile:1]: cya

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+detect-secrets >= 1.2
```
Comment (pvinis) [inline requirements.txt:1]: new version

**Review by artsyjian — COMMENTED** [review]
lgtm

**Comment by brainbicycle** [comment]
this looks good to me, thanks for the fix!
I think long term if we can I would prefer to use the brew version just to avoid having python + pip as another requirement for running Eigen since we already have quite a few requirements to run but curious what others think.

**Review by brainbicycle — APPROVED** [review]

**Comment by brainbicycle** [comment]
it is a fair point but when brew installs python I don't have to think about it for the most part is my experience, and some of the extra control might come at a cost, e.g issues with different versions of python. Either way an issue for another day.

**Pull request merged**